installer/generate_installer.sh

```diff
@@ -93,7 +101,11 @@ if [ ! "$NO_BUILD" ]; then
     # Ensure we have the latest images
     echo "The latest images will be pulled from DockerHub for this build."
     $SUDO docker-compose pull $DOCKER_PULL_SERVICES
-    $SUDO docker-compose build --pull $NO_CACHE $DOCKER_BUILD_SERVICES
+    for version in "${elk_versions[@]}"; do
+      v=$(echo $version|tr -d '\n')
+      #export ELK_STACK_VERSION="$v"
```

Comment:
Please uncomment this line. It appears that commenting this line prevents docker-compose from labeling the resulting elastic and kibana images with the appropriate version tags.

Consider using the docker-compose.yml syntax for specifying the build-args for each of the containers individually instead of passing the build arg as a flag on the next line. 

Example which passes the ELK_STACK_VERSION environment variable in as a build argument:
```
build:
      context: ./elasticsearch
      args:
        ELK_STACK_VERSION: ${ELK_STACK_VERSION} 
```

Reply:
Uncommented the line. I did not convert to docker-compose syntax because it doesn't seem to tag the images as expected. I don't have the bandwidth to look into it at the moment, especially since the current tagging implementation isn't broken.